src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs

```diff
@@ -610,7 +610,7 @@ internal override void OnItemClicked(int clickedIndex, VirtualKeyModifiers modif
 			void SingleSelectionCase()
 			{
 
-				if (ItemsSource is ICollectionView collectionView)
+				if (ItemsSource is ICollectionView collectionView and not ISelectionInfo)
```

Comment:
Please add some short // comment above this to explain why this change was required